datamodel/low/base/schema_proxy.go

```diff
@@ -54,10 +54,12 @@ type SchemaProxy struct {
 }
 
 // Build will prepare the SchemaProxy for rendering, it does not build the Schema, only sets up internal state.
-func (sp *SchemaProxy) Build(root *yaml.Node, idx *index.SpecIndex) error {
-	sp.vn = root
+// Key maybe nil if absent.
+func (sp *SchemaProxy) Build(key, value *yaml.Node, idx *index.SpecIndex) error {
```

Comment:
This looks fine, it's a breaking change however, so we will need to bump to `v0.10`

Reply:
It is indeed breaking change, though i'm not sure how many out there are using Build API. I know we do, but for us it's no issue to change.